test/kafka-ssl/multi.td

```diff
@@ -66,6 +66,47 @@ a
 1
 2
 
+> CREATE SECRET ssl_key_location_kafka AS '/share/secrets/materialized-kafka.key'
```

Comment:
We'll have to figure out (in another PR) how to pivot these to be the actual contents of the key rather than the filepath, since users won't be able (by design) to upload files in cloud.